deps/src/libxml2-2.9.1/doc/xmllint.xml

```diff
@@ -105,7 +105,7 @@
 			<arg choice="plain"><option>--catalogs</option></arg>
 			<arg choice="plain"><option>--nocatalogs</option></arg>
 			<arg choice="plain"><option>--auto</option></arg>
-			<arg choice="plain"><option>--xinclude</option></arg>
+			<arg choice="plain"><option>--xinclude </option></arg>
```

Comment:
Is this change intentional?

Reply:
In general it looks like lots of "include<" became "include <" -- this doesn't seem right for non-source-code files.